### DISCUSSION TITLE
权限不够

### DISCUSSION THREAD

**Discussion by junyuan-fang** [huan/docker-wechat#135] [General]
“[DoChat] 盒装微信 v0.14.4
Disabling patch for /home/user/.wine/drive_c/users/user/AppData/Roaming/Tencent/WeChat ...
Disabling patch for /home/user/.wine/drive_c/users/user/Application Data/Tencent/WeChat ...
mkdir: 无法创建目录 “/home/user/.wine/drive_c/users/user/Application Data/Tencent”: 权限不够“

“root@lx1-fuxi046:/home# “

**Comment by junyuan-fang** [comment]
跑“curl -sL https://raw.githubusercontent.com/huan/docker-wechat/master/dochat.sh | bash” 出现的 "mkdir" 报错

**Comment by YalandHong** [comment]
Duplicate of https://github.com/huan/docker-wechat/issues/136#issuecomment-915774160
